github/resource_github_actions_runner_group.go

```diff
@@ -118,6 +118,8 @@ func resourceGithubActionsRunnerGroupCreate(d *schema.ResourceData, meta interfa
 	d.Set("selected_repositories_url", runnerGroup.GetSelectedRepositoriesURL())
 	d.Set("visibility", runnerGroup.GetVisibility())
 	d.Set("selected_repository_ids", selectedRepositoryIDs) // Note: runnerGroup has no method to get selected repository IDs
+	d.Set("restricted_to_workflows", runnerGroup.GetRestrictedToWorkflows())
```

Comment:
These additional properties should be identified in the appropriate docs page for this resource.